src/main/groovy/de/dkfz/roddy/execution/jobs/cluster/pbs/PBSJobManager.groovy

```diff
@@ -167,165 +57,12 @@ class PBSJobManager extends ClusterJobManager<PBSCommand> {
         return 4
```

Comment:
Is this actually a column index? Maybe rename to `getColumnIndexOfJobID` or `getJobIdColumnIndex`. Same for similar methods.

Reply:
done